### PR TITLE
Update to django 1.8

### PR DESCRIPTION
**Pull request by KevinEtienne**
The library is almost compatible with Django 1.8.
Tests now require migrations otherwise `ProgrammingError` is raised with the following message: `relation "auth_group" does not exist`.
`last_login` default to `None` instead of the current time when the user is created.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+## v12.0.0 (upcoming)
+
+* Add `django v.18` support.
```
Comment (meshy) [inline CHANGELOG.md]: v1.8

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
+## v12.0.0 (upcoming)
+
+* Add `django v1.8` support.
+
```
Comment (meshy) [inline CHANGELOG.md]: As far as I can see this is backwards compatible, so should probably be `11.1.0`

**Comment by meshy** [comment]
LGTM HWTI

**Comment by KevinEtienne** [comment]
Thank you :)

File: .travis.yml [hunk 3]
```diff
@@ -5,10 +5,13 @@ env:
   matrix:
     - TOX_ENV=py27-django16
     - TOX_ENV=py27-django17
+    - TOX_ENV=py27-django18
     - TOX_ENV=py33-django16
     - TOX_ENV=py33-django17
+    - TOX_ENV=py33-django18
     - TOX_ENV=py34-django16
```
Comment (Ian-Foote) [inline .travis.yml]: Can/should we drop django 1.6 now?

Comment (KevinEtienne) [inline .travis.yml]: Not sure about this one

Comment (meshy) [inline .travis.yml]: No need to drop it now. If we need to implement something that requires a shim, then we can drop it then.

File: user_management/api/tests/test_views.py [hunk 4]
```diff
@@ -62,9 +62,8 @@ def test_post(self):
     def test_post_last_login_updates(self):
         """Authenticating updates the user's last_login."""
         user = UserFactory.create(email=self.username, password=self.password)
-        now = timezone.now()
-        self.assertLess(user.last_login, now)
```
Comment (Ian-Foote) [inline user_management/api/tests/test_views.py]: Should we `assertIsNone` now?

Comment (KevinEtienne) [inline user_management/api/tests/test_views.py]: `None` is only valid under django1.8, before the value of last_login is `now`

Comment (Ian-Foote) [inline user_management/api/tests/test_views.py]: The other possibility is to explicitly set `last_login` to a suitable datetime when creating the `user`.

Comment (KevinEtienne) [inline user_management/api/tests/test_views.py]: Good idea, thanks I'll try that

**Comment by meshy** [comment]
This `last_login` logic in the test seems wrong to me. I don't think the test should override the expected behaviour, as it's not really part of the test, and seems a bit redundant. You're setting something, just to then assert that it's been set.

**Comment by KevinEtienne** [comment]
@meshy I'll remove the assert. In fact `last_login` set to `None` or `now` shouldn't be tested as it's part of django

**Comment by meshy** [comment]
:+1:

**Comment by Ian-Foote** [comment]
Yeah, I was really thinking that we should test that the value has actually changed. Maybe:
```python
old_value = user.last_login
request = self.create_request('post', auth=False, data=self.data)
self.view_class.as_view()(request)
user = User.objects.get(pk=user.pk)
self.assertGreater(user.last_login, now)
self.assertNotEqual(user.last_login, old_value)
```

**Pull request merged**